client/dashboard/sites/performance/index.tsx

```diff
@@ -31,6 +35,13 @@ const getPageFromID = ( pages: SitePerformancePage[] | undefined, pageId: string
 };
 
 function SitePerformanceContent( { site }: { site: Site } ) {
+	const { data: siteSettings } = useSuspenseQuery( {
+		...siteSettingsQuery( site.ID ),
+		select: ( s ) => ( {
+			gmtOffset: typeof s?.gmt_offset === 'number' ? s.gmt_offset : 0,
```

Comment:
I think I understand now. My site's `timezoneString` is empty string (not sure why) but it has `gmtOffset` after I updated the timezone in Settings -> General. But the type is string, so this logic will always set it to 0 🥲 

<img width="809" height="580" alt="Image" src="https://github.com/user-attachments/assets/23803ef4-5c2e-49c8-b768-c65a3952f63f" />